src/Todo.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import TodoItem from './TodoItems';
+
+function TodoList(props) {
+  return (
+    <table>
+      <tr>
+        <th>ID</th>
+        <th>TITLE</th>
+        <th>Completed</th>
+        <th>USER</th>
+      </tr>
+      {props.todos.map(item => (
```

Comment:
```suggestion
      {todos.map(item => (
```
если деструктуризировать в пятой строке todos, не надо будет перед ним писать props.